Contribute the handwritten-simplified-chinese recognition-0001 model (#1161)
openvinotoolkit/open_model_zoo: models/intel/handwritten-simplified-chinese-recognition-0001/description/handwritten-simplified-chinese-recognition-0001.md

# handwritten-simplified-chinese-recognition-0001

## Use Case and High-Level Description

This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
reshape layer and a fully connected layer.
The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).

## Example

![](./test.png) -> 的人不一了是他有为在责新中任自之我们

## Specification

| Metric                                         | Value              |
|------------------------------------------------|--------------------|
| GFlops                                         |                    |
| MParams                                        |                    |
| Accuracy on partial SCUT-EPT test set          |   75.58%           |
| Source framework                               | PyTorch\*          |

## Accuracy Values

This demo adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.

## Inputs

> Shape: [1x1x96x2000] - An input image in the format [BxCxHxW],

Reviewer: Please, add layer name. Also reorder to 
```
<I/O type> <layer name> <shape> <format>
```
e.g. [1](https://github.com/opencv/open_model_zoo/blob/master/models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md#input), [2](https://github.com/opencv/open_model_zoo/blob/master/models/public/midasnet/midasnet.md#output)

Reviewer: Here:
```
Gray-scale image, name - <layer name>, shape - [1x1x96x2000], format is [BxCxHxW]
```

where:
  - B - batch size
  - C - number of channels
  - H - image height
  - W - image width

> Note that the source image will be converted to grayscale, resized to specific height (such as 96) while keeping aspect ratio, and right-bottom padded.

Reviewer: 1. "Grayscale" is moved above.
2. If you really need to emphasize the importance of resize options (also you miss the fill value), make it a real note:
> **NOTE:** Image must be resized ...
```
> **NOTE:** Image must be resized ...
```
In other case - delete this sentence completely.

## Outputs

> The net outputs a blob with the shape [125, 1, 4059] in the format [WxBxL], where:

Reviewer: ```suggestion
The net outputs a blob with the shape [125x1x4059] in the format [WxBxL], where:
```
Or use commas as separators everywhere

  - W - output sequence length
  - B - batch size
  - L - confidence distribution across the supported symbols in [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).

> The network output can be decoded by CTC Greedy Decoder.

Reviewer: ```suggestion

The network output can be decoded by CTC Greedy Decoder.
```

# handwritten-simplified-chinese-recognition-0001

## Use Case and High-Level Description

This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
reshape layer and a fully connected layer.

> The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).

Reviewer 2: ```suggestion
The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release) dataset.
```

## Example

![](./test.png) -> 的人不一了是他有为在责新中任自之我们

## Specification

| Metric                                         | Value              |
|------------------------------------------------|--------------------|
| GFlops                                         |                    |

> | MParams                                        |                    |

Reviewer 2: Are you going to fill these in?

Author: For the previous Japanese model, Vladimir helped fill in these values. I'm not sure how I should compute these values.

Reviewer 2: You can use the Model Analyzer, which you can find in the `deployment_tools/tools/workbench/model_analyzer` directory within OpenVINO toolkit.

Author: Thanks for the reply. Should I use FP32 IR or FP16 IR when using the model_analyzer?

Reviewer 2: I don't think it will matter. But FP32, I guess.

| Accuracy on partial SCUT-EPT test set          |   75.58%           |
| Source framework                               | PyTorch\*          |

## Accuracy Values

This demo adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.

## Inputs

Grayscale image, name - `actual_input`, shape - [1x1x96x2000], format is [BxCxHxW], where:
  - B - batch size
  - C - number of channels
  - H - image height
  - W - image width

> **NOTE:**  the source image should be resized to specific height (such as 96) while keeping aspect ratio, and the width after resizing should be no larger than 2000 and then the width should be right-bottom padded to 2000 with edge values.

## Outputs

> Name - `outpput`, shape - [125x1x4059], format is [WxBxL], where:

Reviewer: Is "out**pp**ut" as correct name?

Author: sorry, it was a typo.

# handwritten-simplified-chinese-recognition-0001

## Use Case and High-Level Description

This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
reshape layer and a fully connected layer.
The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release) dataset.

## Example

![](./test.png) -> 的人不一了是他有为在责新中任自之我们

## Specification

| Metric                                         | Value              |
|------------------------------------------------|--------------------|
| GFlops                                         |  134.513        |
| MParams                                        |   17.270           |
| Accuracy on SCUT-EPT test subset (excluding images wider than 2000px after resized to height 96px with aspect ratio)                               |   75.31%           |

> | Accuracy on SCUT-EPT test set of the original PyTorch\* model                |   76.16%           |

Reviewer 3: which one metric we should use for reference?

| Source framework                               |   PyTorch\*        |

## Accuracy Values

> This demo adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.

Reviewer 2: ```suggestion
This model adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.
```

# handwritten-simplified-chinese-recognition-0001

## Use Case and High-Level Description

This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
reshape layer and a fully connected layer.
The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).

## Example

![](./test.png) -> 的人不一了是他有为在责新中任自之我们

## Specification

| Metric                                         | Value              |
|------------------------------------------------|--------------------|
| GFlops                                         |                    |
| MParams                                        |                    |
| Accuracy on partial SCUT-EPT test set          |   75.58%           |
| Source framework                               | PyTorch\*          |

## Accuracy Values

This demo adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.

## Inputs

> Grayscale image, name - `actual_input`, shape - [1x1x96x2000], format is [BxCxHxW],
> 
> where:

Reviewer: ```suggestion
Grayscale image, name - `actual_input`, shape - [1x1x96x2000], format is [BxCxHxW], where:
```

  - B - batch size
  - C - number of channels
  - H - image height
  - W - image width

> **NOTE:**  the source image should be resized to specific height (such as 96) while keeping aspect ratio, and the width after resizing should be no larger than 2000 and then the width should be right-bottom padded to 2000 with edge values.

## Outputs

> The net outputs a blob with the shape [125x1x4059] in the format [WxBxL], where:

Reviewer: Add output layer name, as for input.
